pootle/apps/pootle_store/models.py

```diff
@@ -2212,9 +2210,15 @@ def _get_last_updated(self):
     def _get_last_action(self, submission=None):
         if submission is None:
             try:
-                sub = Submission.simple_objects.filter(store=self) \
-                                .exclude(type=SubmissionTypes.UNIT_CREATE) \
-                                .latest()
+                sub = self.submission_set.select_related(
+                    'unit',
+                    'unit__store',
```

Comment:
I'm not sure. I think sub.unit.store is used. I'll check one more time.
Actually there exists a workaround:
```
sub.store = self
sub.unit.store = self
```